src/core/bridge.ts

```diff
@@ -19,6 +20,13 @@ export interface BridgeCallback {
   (err?: Error, res?: Results): void;
 }
 
+export class BridgeTimeout extends Error {
+  constructor (err: string) {
+    super(err);
+    Object.setPrototypeOf(this, BridgeTimeout.prototype);
```

Comment:
Why are we calling this?

Reply:
Tests were failing before when I didn't have it. Now they're not. I guess I'll just remove it.